modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationMetricsListener.java

```diff
@@ -48,6 +51,9 @@
     /** Sent bytes count.*/
     private final LongAdder sentBytesCnt = new LongAdder();
 
+    /** Metric registry. */
+    MetricRegistry mreg;
```

Comment:
We should cache specific counters in the constructor instead of querying it from a registry on each access.